core/controllers/topic_editor_test.py

```diff
@@ -145,6 +148,43 @@ def test_get(self):
                 ), expected_status_int=401)
             self.logout()
 
+    def test_get_questions_with_multi_skills(self):
+        for i in range(0, 4): #pylint: disable=unused-variable
```

Comment:
Avoid adding pylint-disable. Use _ instead of i.

Reply:
Done.